give/src/components/Title/Title.js

```diff
@@ -0,0 +1,70 @@
+// Based on v3
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/title.html
+// https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/title.html
+
+import "./Title.css";
+
+const LinkWrapper = ({ href, children }) => {
```

Comment:
Is this link wrapper elsewhere? Sounds like it should be its own component

Reply:
Yeah, I was thinking that same thing. I've created some version of this in three of the four components I've overhauled recently.

Reply:
https://github.com/CruGlobal/cru-content-designs/pull/8/commits/88e6912290a450091d57cf2aab195aaa31953570